### PR TITLE
Unify tooltip appearance and add to new docs icon and Veteran ID copy button

### PR DESCRIPTION
**Pull request by lowellrex**
Connects #6370.

* Unifies the styling of tooltips
* Adds tooltip to new documents icon
* Adds tooltip to Veteran ID copy button

Additionally, this PR creates a new `Tooltip` component which wraps `ReactTooltip`, requires a text attribute (the text of the tooltip) and a child element (the element that triggers the tooltip to appear when it is hovered).

### New documents tooltip
| Table view | Case details title |
| --- | --- |
| image | image |

### Column header tooltip
| Before | After |
| --- | --- |
| image | image |

### Veteran ID copy button tooltip
image

### CODE REVIEW

File: client/app/queue/components/NewFile.jsx [hunk 1]
```diff
@@ -19,6 +22,17 @@ type Props = Params & {|
...
 
+const tooltipID = 'newfile-tip';
+
+const tooltipStyling = css({
+  [`& > #${tooltipID}`]: {
+    backgroundColor: COLORS.GREY_DARK,
+    padding: '0.5rem 1rem',
+    textAlign: 'center'
+  },
+  [`& > #${tooltipID}:after`]: { display: 'none' }
```
Comment (lowellrex) [inline client/app/queue/components/NewFile.jsx]: Removes the caret from the bottom of the tooltip.

Comment (mdbenjam) [reply client/app/queue/components/NewFile.jsx]: Is it important to remove the caret? I know the mocks don't have it, but we keep the caret on our task table. I'd like to be consistent here. If we decide there shouldn't be a caret then can we wrap `ReactTooltip` in our own component with these changes? I don't want every usage to have to specify its own CSS.

Comment (mdbenjam) [reply client/app/queue/components/NewFile.jsx]: The same goes for the text align and color.

**Comment by mdbenjam** [comment]
There is one other place we use the react tooltip: https://github.com/department-of-veterans-affairs/caseflow/blob/master/client/app/hearings/components/WorksheetHeader.jsx#L140-L156 We could try to unify that as well... Don't think that's necessary with this PR though.

**Pull request merged**